### DISCUSSION TITLE
XCH tokens did not come to the CHIA light wallet

### DISCUSSION THREAD

**Discussion by RAkot13** [Chia-Network/chia-blockchain#14332] [Support]
Hello, I ran into   a problem: when withdrawing XCH from the MEXC exchange, 2,181 XCH tokens got   stuck, the withdrawal was made on December 23, 2022, the exchange transferred   xch, There is a Hash Transaction with withdrawal confirmation, How to solve   this problem? Installed CHIA light wallet, synchronization does not start.

https://xchscan.com/zh/txns/0x78e344cb8311ef496aaa050d367145f1ca6a9bbd48596cbda726c9770570d6a2

**Comment by Finitossssss** [comment]
Seems situation. I need help

**Comment by Bodman1** [comment]
this transaction was made on January 18, 2023, 7:22 am, at 33 chia, that evening I also transferred 27.9482 chia kucoin from the exchange to my wallet, in the support of the exchange they wrote to me that the coins were sent to the wallet, but on the wallet I didn't see them. transaction data

The withdrawal you made on 2023-01-19 03:04:26 (UTC+08:00) was successful.
Token: | XCH
Amount: | 27.9482
Address: | xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72
